src/App.tsx

```diff
@@ -48,6 +48,7 @@ const App = () => {
 
   const {data: evaluatedSchema, error: schemaError} = zod.validateSchema(schema)
 
+  // biome-ignore lint/correctness/useExhaustiveDependencies: isLoading is not a dependency
```

Comment:
This shouldn't be here. Can you provide a reason why you added this?

Reply:
Didn't we just agree to remove the isLoading check? :confused:

Reply:
Yeah, if you do so, you can remove this rule exclusion, right?

Reply:
By removing the check I mean to go from

If(isLoading || !Monaco)

To this

If(!Monaco)


It's all written and explained in the above comments